qemu/tests/check_coredump.py

```diff
@@ -148,51 +148,69 @@ def run(test, params, env):
     4) If yes, copy them to working directory.
     """
 
+    error_msgbox = []
+    error.context("Check crash/coredump in test result dir", logging.info)
+    root_dir = os.path.dirname(os.path.dirname(test.debugdir))
+    crash_dir = []
+    for prefix in ["crash", "core"]:
+        pattern = "%s/*/%s.*" % (root_dir, prefix)
+        crash_dir += glob.glob(pattern)
+    if crash_dir:
+        crash_msg = ("Crash/Coredump happened, " +
+                     "Please check dump files under %s" % crash_dir)
```

Comment:
The result is not friendly enough when there are more than one cores:
1. Suggest to add core num prompt at the end of test, e.g "There are xx cores found in test"
2. Can use use casename/core_name instead of path of core_name, as it's a little strange when it's not run in local e.g virt.qemu.smp_2.2048m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.7.1.x86_64.io-github-autotest-qemu.boot/core.kvm.1111 or boot/core.kvm.1111
3. Separate the cases in the list, e.g "\n" or other method.

Result when there are two cores currently.
['/root/autotest-devel/client/results/default/virt.qemu.smp_2.2048m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.7.1.x86_64.io-github-autotest-qemu.boot/core.kvm.1111', '/root/autotest-devel/client/results/default/virt.qemu.smp_2.2048m.repeat1.run_test.Host_RHEL.7.1.qcow2.virtio_blk.up.virtio_net.RHEL.7.1.x86_64.io-github-autotest-qemu.shutdown/core.kvm.1111']